### PR TITLE
chore: remove /* GraphQL */ from screen templates #165

### PR DESCRIPTION
**Pull request by ilya-trifonov**
affects: @amplicode/codegen

### CODE REVIEW

File: example-app/src/app/screens/standalone-list/StandaloneOwnerCards.tsx [hunk 1]
```diff
@@ -16,26 +16,30 @@ import { getOwnerDTODisplayName } from "../../../core/display-name/getOwnerDTODi
...
-    delete_Owner(id: $id)
-  }
 `);
 
+const DELETE__OWNER = gql(`
+    
+mutation Delete_Owner($id: BigInteger!) {
+  delete_Owner(id: $id)
+}
+
```
[the remark this answers is not in the document]
Comment (ilya-trifonov) [reply example-app/src/app/screens/standalone-list/StandaloneOwnerCards.tsx:40]: @vyacheslav-pushkin 
empty lines are added after the screens are generated. Syntax highlighting in these queries also disappeared. Everything works fine if we don`t remove /* GraphQL */

[the remark this answers is not in the document]
Comment (vyacheslav-pushkin) [reply example-app/src/app/screens/standalone-list/StandaloneOwnerCards.tsx:40]: Where the empty lines are coming from?

[the remark this answers is not in the document]
Comment (vyacheslav-pushkin) [reply example-app/src/app/screens/standalone-list/StandaloneOwnerCards.tsx:40]: Which version of Studio are you using?

[the remark this answers is not in the document]
Comment (ilya-trifonov) [reply example-app/src/app/screens/standalone-list/StandaloneOwnerCards.tsx:40]: I use npm run-bootstrap-react-app to generate screens.

File: example-app/src/app/screens/owner/OwnerCards.tsx [hunk 2]
```diff
@@ -18,25 +18,25 @@ import { getOwnerDTODisplayName } from "../../../core/display-name/getOwnerDTODi
...
-      email
+      ownerList {
+        id
+        firstName
+        lastName
+        city
+        address
+        telephone
+        email
+      }
```
Comment (vyacheslav-pushkin) [inline example-app/src/app/screens/owner/OwnerCards.tsx:31]: Extra whitespace appeared

**Review by vyacheslav-pushkin — CHANGES_REQUESTED** [review]

**Review by vyacheslav-pushkin — APPROVED** [review]

**Comment by vyacheslav-pushkin** [comment]
@ilya-trifonov please resolve conflicts and merge

**Pull request merged**